vars/MPLPipelineConfig.groovy

```diff
@@ -41,7 +43,14 @@ def call(body, Map defaults = [:], Map overrides = [:]) {
   if( body in Closure ) {
     body.resolveStrategy = Closure.DELEGATE_FIRST
     body.delegate = config
+
+    body.setProperty('env', env)
+    body.setProperty('params', params)
+    body.setProperty('currentBuild', currentBuild)
     body()
+    config.remove('env')
```

Comment:
Maybe we can setProperty&remove for "config" variable? I suppose behavior will be same.
Otherwise it looks strange. If not, please add comment.
Due to our delegation strategy previous setProperty actually set property for "config" variable.

Reply:
Yep, changed to modify the config itself.